nfs/cmd/nfs-provisioner/main.go

```diff
@@ -42,6 +42,7 @@ var (
 	gracePeriod    = flag.Uint("grace-period", 90, "NFS Ganesha grace period to use in seconds, from 0-180. If the server is not expected to survive restarts, i.e. it is running as a pod & its export directory is not persisted, this can be set to 0. Can only be set if both run-server and use-ganesha are true. Default 90.")
 	enableXfsQuota = flag.Bool("enable-xfs-quota", false, "If the provisioner will set xfs quotas for each volume it provisions. Requires that the directory it creates volumes in ('/export') is xfs mounted with option prjquota/pquota, and that it has the privilege to run xfs_quota. Default false.")
 	serverHostname = flag.String("server-hostname", "", "The hostname for the NFS server to export from. Only applicable when running out-of-cluster i.e. it can only be set if either master or kubeconfig are set. If unset, the first IP output by `hostname -i` is used.")
+	exportSubnet   = flag.String("export-subnet", "*", "Subnet for NFS export to allow mount only from")
```

Comment:
we should add a note that it has no effect if *useGanesha is true. Although we could enable it for ganesha as well someday.

Some validation code to check for the conflict "useGanesha is true but exportSubnet is non-default" could also be nice but I don't mind not having it.

Reply:
Thanks! I added warning when using useGanesha together with exportSubnet, I adjusted also some test cases, however one of test job is failing and I don't know why.